rust/kcl-lib/src/execution/mod.rs

```diff
@@ -2244,7 +2244,7 @@ w = f() + f()
     |> line(end = [0, 0])
     |> close()
 }
-  
+
```

Comment:
what the fuck is this?

`cargo fmt` does nothing

Reply:
It's inside a Rust string, so cargo fmt shouldn't modify your strings.